solr/views.py

```diff
@@ -165,9 +177,9 @@ def post(self):
                 {'error': 'You can only pass one content stream.'}), 400
 
         if 'fq' not in query:
-            query['fq'] = [u'{!bitset}']
-        elif len(filter(lambda x: '!bitset' in x, query['fq'])) == 0:
-            query['fq'].append(u'{!bitset}')
+            query['fq'] = '{!bitset}'
+        elif '{!bitset}' not in query['fq']:
+            query['fq'] = ",".join(query['fq'].split(",") + [u'{!bitset}'])
```

Comment:
i'm slightly confused by this; `{!....}` is usually used as a prefix; in the case of a bigquery it stands alone `fq={!bitset}` but I *think* this is not going to work: `title:redshift,{!bitset}`

at least that would be a big surprise (for me) if it did :)